edgelet/edgelet-http/tests/systemd.rs

```diff
@@ -2,22 +2,16 @@
 #![deny(unused_extern_crates, warnings)]
```

Comment:
Replace  `unused_extern_crates` with `rust_2018_idioms` here as well, and then you should be able to just use lazy_static.